build/copy.js

```diff
@@ -24,39 +24,20 @@ module.exports = function (grunt) {
                 { expand: true, cwd:  'src/root_files/app', src: ['index.html'],        dest: global.dist },
 
                 // app, static
-                { expand: true, cwd: 'src/download/',                  src: ['**'],                               dest: `${global.dist}/download/` },
-                { expand: true, cwd: 'src/images/',                    src: ['**', '!app_2/**', '!common/logos'], dest: `${global.dist}/images/` },
-                { expand: true, cwd: 'src/javascript/_autogenerated/', src: ['*.js'],                             dest: `${global.dist}/js/texts/` },
-                { expand: true, cwd: 'src/javascript/',                src: ['manifest.json'],                    dest: global.dist },
+                { expand: true, cwd: 'src/download/',                  src: ['**'],                  dest: `${global.dist}/download/` },
+                { expand: true, cwd: 'src/images/',                    src: ['**', '!common/logos'], dest: `${global.dist}/images/` },
+                { expand: true, cwd: 'src/javascript/_autogenerated/', src: ['*.js'],                dest: `${global.dist}/js/texts/` },
+                { expand: true, cwd: 'src/javascript/',                src: ['manifest.json'],       dest: global.dist },
 
                 // binary-style
                 { expand: true, cwd: `${global.node_modules_paths.binary_style}/src/images/favicons`, src: ['**'], dest: `${global.dist}/images/favicons/` },
                 { expand: true, cwd: `${global.node_modules_paths.binary_style}/src/images/logo`,     src: ['**'], dest: `${global.dist}/images/logo/` },
             ],
         },
-        app_2: {
-            files: [
-                ...common,
-                { expand: true, cwd:  'src/root_files/app_2', src: ['sitemap*.xml'],      dest: 'dist' }, // top level
-                { expand: true, cwd:  'src/root_files/app_2', src: ['index.html'],        dest: global.dist_app_2 },
-
-                // app_2
-                { expand: true, cwd: 'src/images/app_2',                     src: ['**'],   dest: `${global.dist_app_2}/images/` },
-                { expand: true, cwd: 'src/javascript/_autogenerated/app_2/', src: ['*.js'], dest: `${global.dist_app_2}/js/texts/` },
-
-                // binary-style
-                { expand: true, cwd: `${global.node_modules_paths.binary_style}/src/images/favicons`, src: ['**'], dest: `${global.dist_app_2}/images/favicons/` },
-
-                // ChartIQ dependencies
-                { expand: true, cwd: `${global.node_modules_paths.smartcharts}/dist/`, src: ['*.smartcharts.*'],  dest: `${global.dist_app_2}/js/smartcharts/` },
-                { expand: true, cwd: `${global.node_modules_paths.smartcharts}/dist/`, src: ['smartcharts.css*'], dest: `${global.dist_app_2}/css/` },
-            ],
-        },
         get all() {
```

Comment:
I don't think we need all() here either since it's only one section now? Same in the files below

Reply:
I kept it in case we want to maintain the config to add sections again in the future. should I remove it?

Reply:
talked to ashkan, will remove it for now